app/templates/suppliers/create_company_summary.html

```diff
@@ -41,50 +41,75 @@
 
   <h1 class="govuk-heading-l">Check your information </h1>
 
-  {{ summary.heading('Your company details') }}
-  {% call summary.mapping_table(
-    caption="Your company details",
-    field_headings=[
-      "Label",
-      "Value",
-    ],
-    field_headings_visible=False
-  ) %}
-
-  {% set required_company_fields = [
-      ("duns_number", "DUNS number", url_for(".duns_number", _anchor="duns_number")),
-      ("company_name", "Company name", url_for(".company_details", _anchor="company_name")),
-      ("contact_name", "Contact name", url_for(".company_details", _anchor="contact_name")),
-      ("email_address", "Contact email", url_for(".company_details", _anchor="email_address")),
-      ("phone_number", "Contact phone number", url_for(".company_details", _anchor="phone_number"))
-    ]
-  %}
-
-  {% for field, label, url in required_company_fields %}
-    {% call summary.row(complete=session.get(field)) %}
-      {{ summary.field_name(label) }}
-      {{ summary.text(session.get(field, "You must answer this question.")) }}
-      {{ summary.edit_link("Edit", url, hidden_text=label) }}
-    {% endcall %}
-  {% endfor %}
-
-  {% endcall %}
+  <h2 class="govuk-heading-m">Your company details</h2>
+  {{ govukSummaryList({
+    "rows": [{
+      "key": {"text": "DUNS number"},
+      "value": {"text": session.get("duns_number", "You must answer this question.")},
+      "actions": {
+        "items": [{
+          "href": url_for(".duns_number", _anchor="duns_number"),
+          "text": "Edit",
```

Comment:
I would like to see us be closer to the Design System wherever possible; could we change the text here and for other links to "Change"?

```suggestion
          "text": "Change",
```

Reply:
Oh cool sure.